README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
 | multi-line diffing                                                                        | ✔️       | ✖️                                                           | ✖️                                                                |
 | Contextual failures                                                                       | ✔️       | ✖️                                                           | ✖️                                                                |
-| Inter-test progress output during test suite execution                                    | ✔️       | ✔️                                                           | ?                                                                |
-| Intra-test progress output during test suite execution                                    | 🚧      | ✔️                                                           | ?                                                                |
+| Inter-test progress output during test suite execution                                    | ✔️       | ✔️                                                           | ✔️                                                                |
+| Intra-test progress output during test suite execution                                    | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-quickcheck)      |
 | Optional standard output and standard error suppression [2]                               | ✖️       | ✖️                                                           | ✖️                                                                |
-| Aquire and release a resource for every test in a group (`before` and `after`)            | ✔️       | ✔️                                                           | ✖️                                                                |
+| Aquire and release a resource for every test in a group (`before` and `after`)            | ✔️       | ✔️                                                           | C                                                                |
 | Aquire and release a resource once for an entire test group (`beforeAll` and `afterAll`)  | ✔️       | ✔️                                                           | ✔️                                                                |
 | Wrap a single test to use a `withResource`-like function (`around`)                       | ✔️       | ✔️                                                           | ✖️                                                                |
 | Wrap a test group to use a `withResource`-like function (`aroundAll`)                     | ✔️       | ✖️                                                           | ✖️                                                                |
```

Comment:
You said this was possible, but `withResource` does not allow for it because `withResource` requires you to `acquire` and `release` the resource seperately. It doesn't allow you to pass in a `(a -> IO ()) -> IO ()` function.

Reply:
Sorry, I'm not quite sure I understand what you mean here. Can you expand on what it is you want to do? Then I can probably say what the `tasty` equivalent would be, if there is one.

Reply:
`withResource` works with a `acquire` and `release` type resource, but not for something like https://hackage.haskell.org/package/warp-3.3.13/docs/Network-Wai-Handler-Warp.html#v:testWithApplication I think. Is that right?